inssqs/helper.go

```diff
@@ -0,0 +1,37 @@
+package inssqs
+
+import (
+	"encoding/json"
+)
+
+func createBatches[T any](v []T, recordLimit int, byteLimit int) ([][]T, error) {
```

Comment:
It'be great reusable function in a seprate package such as collectionhelper, inskinesis also used something similar.

Reply:
`insdash` mi yoksaa

Reply:
`insdash` would be great if there's no fitting slice library already available. For instance, we're also frequently needing `slice.Contains` which is also lacking in Go stdlib, which we'd expect such a library to provide.

Reply:
_there will be always not-fitting parts to us on 3rd party libraries..._

I created the `insdash`, we can add our helper functions there, incrementally, when we need.